src/pages/library/DataBrowser.js

```diff
@@ -33,117 +82,181 @@ const sidebarSections = [{
     'Exome',
     'Whole Genome'
   ]
+}, {
+  name: 'File type',
+  labels: [
+    'Rds', 'Robj',
+    'bam', 'csv', 'csv.gz', 'fastq', 'fastq.gz',
+    'h5', 'h5ad', 'loom', 'mtx', 'mtx.gz', 'pdf',
+    'rds', 'rds.gz', 'tar', 'tar.gz', 'tsv',
+    'tsv.gz', 'txt', 'txt.gz', 'xlsx', 'zip'
+  ]
 }]
 
+const getRawList = () => new Promise(resolve => setTimeout(() => {
+  fetch('hca-sample.json').then(response => {
+    response.json().then(tempData => {
+      resolve(tempData.data)
+    })
+  })
+}, 1000))
```

Comment:
Could you convert this to async/await? We tend to avoid chaining `.then`s in the codebase. Something like:
```suggestion
const getRawList = async () => {
  const list = await fetch('hca-sample.json').then(res => res.json())
  return new Promise(resolve => setTimeout(resolve(list.data), 1000))
}
```